samples/Ch18_using_libs/fig_18_14_pstl_usm_device.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2023 Intel Corporation
+
+// SPDX-License-Identifier: MIT
+
+#include <oneapi/dpl/execution>
+#include <oneapi/dpl/algorithm>
+#include <sycl/sycl.hpp>
+
+int main() {
+  sycl::queue Q;
+  const int n = 10;
+  int * d_head = static_cast<int *>(
+    sycl::malloc_device(n * sizeof(int),
+                        Q.get_device(),
+                        Q.get_context()));
+
+  std::fill(oneapi::dpl::execution::make_device_policy(Q),
+            d_head, d_head + n, 78);
+  Q.wait();
+
+  sycl::free(d_head, Q.get_context());
```

Comment:
Can just pass the queue here, also.

```suggestion
  sycl::free(d_head, Q);
```